lib/rucio/client/commands/rse.py

```diff
@@ -0,0 +1,268 @@
+# Copyright European Organization for Nuclear Research (CERN) since 2012
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+from typing import TYPE_CHECKING
+
+from rucio.client.commands.bin_legacy.rucio import list_rses
+from rucio.client.commands.bin_legacy.rucio_admin import (
+    add_distance_rses,
+    add_protocol_rse,
+    add_qos_policy,
+    add_rse,
+    del_protocol_rse,
+    delete_attribute_rse,
+    delete_distance_rses,
+    delete_limit_rse,
+    delete_qos_policy,
+    disable_rse,
+    get_attribute_rse,
+    get_distance_rses,
+    info_rse,
+    list_qos_policies,
+    set_attribute_rse,
+    set_limit_rse,
+    update_distance_rses,
+    update_rse,
+)
+from rucio.client.commands.command_base import CommandBase
+
+if TYPE_CHECKING:
+    from argparse import ArgumentParser
+
+    from rucio.client.commands.utils import OperationDict
+
+
+class RSE(CommandBase):
```

Comment:
While aware that naming things is hard, I’m not too enthusiastic about calling this class `RSE`. We have an `RSE` SQLAlchemy model and an `RSE` Flask blueprint.

Then we have `Limit` below but there’s another `Limit` in `account.py`.

I’m requesting that they be changed, just trying to raise awareness about something that could prove suboptimal in retrospect.